go/downstream-test/README.md

```diff
@@ -33,14 +33,14 @@ steps:
   - name: Checkout Upstream
     uses: actions/checkout@v3
     with:
-      path: knative.dev/pkg
+      path: pkg
   - name: Checkout Downstream
     uses: actions/checkout@v3
     with:
       repository: knative-sandbox/sample-controller
-      path: knative.dev/sample-controller
+      path: controller
   - uses: knative/actions/go/downstream-test@main
     with:
-      upstream-module: knative.dev/pkg
-      downstream-module: knative.dev/sample-controller
+      upstream-path: pkg
```

Comment:
The readme gives the example `./knative.dev/pkg` but only `pkg` is here?

Reply:
let me add an update